python/cudf/cudf/utils/gpu.pyx

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
```

Comment:
This should still go in the `_libxx` folder, but lets make a `cuda` module so this can be potentially separated out in the future

Reply:
In view of below comment do we make a `cudf/cudf/cuda` module then?